platform/platform-migratable/src/test/java/org/codice/ddf/platform/migratable/impl/PlatformMigratableTest.java

```diff
@@ -90,13 +92,16 @@ public void testExportValidRelativePaths() throws Exception {
 
         // Verify keystore and truststore are exported
         assertKeystoresExport(mockMigratableUtil);
+        
+        // Verify application list is exported
```

Comment:
Assert method names are pretty clear so I 'm not sure we really need those inline comments.